packages/vitest/src/integrations/snapshot/client.ts

```diff
@@ -26,31 +26,32 @@ interface AssertOptions {
 
 export class SnapshotClient {
   test: Test | undefined
+  suite: Suite | undefined
   snapshotState: SnapshotState | undefined
   snapshotStateMap = new Map<string, SnapshotState>()
 
-  async setTest(test: Test) {
-    this.test = test
+  async setTaskBase(test: Test, suite?: Suite) {
+    suite ? this.suite = suite : this.test = test
```

Comment:
Why do we need a suite here? Snapshots are test specific, they should not rely on a suite.